stratagem-runner/.gitignore

```diff
@@ -0,0 +1,3 @@
+/target
```

Comment:
This file isn't needed, these files are already ignored in the dir one level up.